tutorials/polkadot-sdk/parachains/obtain-coretime.md

```diff
@@ -0,0 +1,117 @@
+- Alternatively, use on-demand coretime to produce blocks as required
+
+## Prerequisites 
+
+Before proceeding, you should have the following items:
+
+- A parachain ID
+- A chain specification
+- A registered parathread with the correct genesis, runtime, and parachain ID that matches the chain specification
+- A properly configured and synced (with the relay chain) collator
+
+!!!note
+    You may only consider these prerequisites if you wish to purchase a core with a parachain.
+
+Once the above is complete, obtaining coretime is the last step to enable your parachain to start producing and finalizing blocks using the relay chain's validator set. If you don't, refer to the previous tutorial: [Deploy on Paseo TestNet](todo).
+
+## Purchasing Bulk Coretime
+
+Purchasing bulk coretime involves purchasing a core from the [Coretime Chain](/polkadot-protocol/architecture/system-chains/coretime/){target=\_blank}, which has an instance of [`pallet_broker`](https://paritytech.github.io/polkadot-sdk/master/pallet_broker/index.html){target=\blank} (the Broker pallet). Although this can be done via sending extrinsics through a tool like Polkadot.js Apps, there are user interfaces for purchasing and managing bulk coretime:
+
+- [RegionX Coretime Marketplace (includes Paseo support)](https://app.regionx.tech){target=\blank}
+- [Lastic Coretime Marketplace](https://www.lastic.xyz/polkadot/bulkcore1){target=\blank}
+  
+!!!info Obtaining a Core on Paseo
+ Obtaining a core for bulk coretime on Paseo follows a different process from Polkadot or Kusama. To apply for a core on Paseo, visit their guide for doing so: [PAS-10 Onboard Paras Coretime](https://github.com/paseo-network/paseo-action-submission/blob/main/pas/PAS-10-Onboard-paras-coretime.md#summary){target=\blank} 
+
+
+### Getting Coretime Funds
+
+First, ensure your wallet is connected to the RegionX interface. To do so, go to Home in the RegionX app and click the Connect Wallet button in the upper right.
+
+After connecting your wallet, you must obtain funds on the Coretime Chain. You can use the [RegionX Transfer](https://app.regionx.tech/transfer){target=\blank} page to perform a cross-chain transfer from the relay to the system chain:
+
+![](/images/tutorials/polkadot-sdk/parachains/obtain-coretime/obtain-coretime-1.webp)
+
+!!!info 
+    If you are purchasing a core on a TestNet, be sure to visit the [Polkadot Faucet](https://faucet.polkadot.io/westend){target=\_blank}.
+
+If successful, you should see the balance in the upper right of the **Transfer** page update with balances on the relay and Coretime chain, respectively: 
+
+![](/images/tutorials/polkadot-sdk/parachains/obtain-coretime/obtain-coretime-2.webp)
+
+### Purchasing a Core
+
+For this tutorial, we will use [RegionX](https://app.regionx.tech){target=\_blank}. Once you open the app, you should be presented with the following screen:
+
+![](/images/tutorials/polkadot-sdk/parachains/obtain-coretime/obtain-coretime-3.webp)
```

Comment:
```suggestion
![Screenshot of the RegionX app displaying the main interface.](/images/tutorials/polkadot-sdk/parachains/obtain-coretime/obtain-coretime-3.webp)
```

In this case since there aren't steps above it that can summarize what is in the image, lets add alt text